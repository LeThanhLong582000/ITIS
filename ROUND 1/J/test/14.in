14
377259208 201975317
983903132 379941297
362512826 746021386
982576274 872808752
856275005 36905960
9753314 656503551
958497677 681884400
443541716 259785992
6432638 941017753
378096927 957532463
363442417 90807450
698544584 462994927
579849143 233566120
907496152 618169916
