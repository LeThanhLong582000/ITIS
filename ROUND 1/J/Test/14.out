3018073664
7871225056
2900102608
7860610192
5993925035
78026512
7667981416
3548333728
51461104
3024775416
2544096919
5588356672
4638793144
7259969216
